nocrypto.opam

```
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+homepage:     "https://github.com/mirleft/ocaml-nocrypto"
+dev-repo:     "git+https://github.com/mirleft/ocaml-nocrypto.git"
+bug-reports:  "https://github.com/mirleft/ocaml-nocrypto/issues"
+doc:          "https://mirleft.github.io/ocaml-nocrypto/doc"
+authors:      ["David Kaloper <dk505@cam.ac.uk>"]
+maintainer:   "David Kaloper <dk505@cam.ac.uk>"
+license:      "ISC"
+synopsis:     "Simple crypto for the modern age"
+
+build: [ ["dune" "subst"] {pinned}
+         ["dune" "build" "-p" name "-j" jobs ]
+         ["dune" "runtest"] {with-test} ]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {build}
```

```suggestion
  "dune" {build & >="1.7"}
```

to match the dune version

---

Added, but I'm not sure what is the minimal dune language version that this needs?